extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItem.java

```diff
@@ -94,4 +95,12 @@ public void setHeartedByUploader(boolean isHeartedByUploader) {
     public boolean getHeartedByUploader() {
         return this.heartedByUploader;
     }
+
+    public boolean getPinned() {
```

Comment:
Ah damn. I was too fast. That should be `isPinned` (and `isHeartedByUploader`)

Reply:
You could just click on `Revert`?

Reply:
No, I'll create a new PR for both

Reply:
https://github.com/TeamNewPipe/NewPipeExtractor/pull/525